recipes/libxml2/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if [[ `uname` == 'Darwin' ]];
+then
+    export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+elif [[ `uname` == 'Linux' ]];
+then
+    export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+fi
+
+./autogen.sh
+./configure --prefix="${PREFIX}" \
+            --with-iconv="${PREFIX}" \
+            --with-icu \
+            --with-lzma="${PREFIX}" \
+            --with-python="${PREFIX}" \
+            --with-zlib="${PREFIX}"
+make
+# Correct weirdly linked library paths.
+if [[ `uname` == 'Darwin' ]];
+then
+    LIBDIR="${PREFIX}/lib"
+    LIBICUDATA="`cd ${LIBDIR} && ls libicudata.*.*.dylib`"
```

Comment:
This make sure we get the correct name of this library even if we use different versions in the future. This is the name the library will have and not a link to it.